cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -27,6 +35,12 @@ def install():
         hookenv.status_set('blocked', 'Missing kubernetes resource')
         return
 
+    # Handle null resource publication, we check if its filesize < 1mb
+    filesize = os.stat(archive).st_size
+    if filesize < 1000000:
+        hookenv.status_set('blocked', 'Missing kubernetes resource')
```

Comment:
I would make this message different so we know the file was not complete. Suggest:
```
The kubernetes resource is not complete.
```